exporter/osg/osgdata.py

```diff
@@ -1452,16 +1469,18 @@ def get_vertex_key(faceindex,facevertexindex):
                     for uv in mesh.tessface_uv_textures:
                         uvs.append(uv.data[face.index].uv[facevertexindex])
 
-                    for vertex_group in  mesh.vertices[vert_index].groups:
-                        inf = [self.object.vertex_groups[vertex_group.group ].name, vertex_group.weight]
-                        #try:getBoneByName(inf[0])   #check bone existence
-                        if  inf[0] != "" and inf[1] > 0.0001:
-                            if inf[0] not in vgroups:
-                                vg = VertexGroup()
-                                vg.targetGroupName = spaceSafe(inf[0] + armature_name)
-                                vg.vertexes.append((newindex, inf[1])) #= (vertex_weight_list
-                                vgroups[inf[0]] = vg
-                            vgroups[inf[0]].vertexes.append((newindex, vertex_group.weight))
+                    if self.object.vertex_groups:
+                        for vertex_group in  mesh.vertices[vert_index].groups:
+                            inf = [self.object.vertex_groups[vertex_group.group ].name, vertex_group.weight]
```

Comment:
`inf` => `influence` I suppose; beware of naming, this is hard to read. We've got `group.group for group in groups` huh